### DISCUSSION TITLE
Kibana 404 page not found after updates

### DISCUSSION THREAD

**Discussion by Axtrus** [Security-Onion-Solutions/securityonion#9299] [General] [answered]
Hi Guys,

I have an issue with my kibana returning 404 after updating my Security Onion device to version 2.3.190 for standalone installation.
I tried several things but nothing works, can anyone know how to fix this problem? you can see more details in the logs on the attached pictures or below.

Thanks in advance

![image](https://user-images.githubusercontent.com/74593771/205898220-2c4414c1-19e8-4afa-b4bc-74549c15434e.png)
![image](https://user-images.githubusercontent.com/74593771/205898433-1f3a396c-7785-470b-aa6f-2434717f83f5.png)

Error in the logs:
**_{"node":{"roles":["background_tasks","ui"]}},"ecs":{"version":"8.4.0"},"@timestamp":"2022-12-06T10:17:14.720+00:00","message":"Server is stopping; must stop all async operations","error":{"message":"Server is stopping; must stop all async operations","type":"Error","stack_trace":"Error: Server is stopping; must stop all async operations\n    at /usr/share/kibana/x-pack/plugins/rule_registry/server/rule_data_plugin_service/resource_installer.js:66:20\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"},"log":{"level":"ERROR","logger":"plugins.ruleRegistry"},"process":{"pid":7},"trace":{"id":"6882ee00e69c840f82753d32207318dd"},"transaction":{"id":"7f594a8f866b59f0"}}
{"service":{"node":{"roles":["background_tasks","ui"]}},"ecs":{"version":"8.4.0"},"@timestamp":"2022-12-06T10:17:14.721+00:00","message":"Failure installing common resources shared between all indices. Server is stopping; must stop all async operations","error":{"message":"Failure installing common resources shared between all indices. Server is stopping; must stop all async operations","type":"Error","stack_trace":"Error: Failure installing common resources shared between all indices. Server is stopping; must stop all async operations\n    at ResourceInstaller.installWithTimeout (/usr/share/kibana/x-pack/plugins/rule_registry/server/rule_data_plugin_service/resource_installer.js:75:13)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at ResourceInstaller.installCommonResources (/usr/share/kibana/x-pack/plugins/rule_registry/server/rule_data_plugin_service/resource_installer.js:89:5)"},"log":{"level":"ERROR","logger":"plugins.ruleRegistry"},"process":{"pid":7},"trace":{"id":"6882ee00e69c840f82753d32207318dd"},"transaction":{"id":"7f594a8f866b59f0"}}
{"service":{"node":{"roles":["background_tasks","ui"]}},"ecs":{"version":"8.4.0"},"@timestamp":"2022-12-06T10:17:14.738+00:00","message":"[.kibana] SET_SOURCE_WRITE_BLOCK -> FATAL. took: 64050ms.","log":{"level":"INFO","logger":"savedobjects-service"},"process":{"pid":7},"trace":{"id":"3411478d49ce00742feeb46db7fd34d4"},"transaction":{"id":"143d11bc7d7870c6"}}_**

**Comment by dougburks** [comment, marked as answer]
What version did you upgrade from?

What are the hardware specs of your system?

Have you reviewed the following?
https://docs.securityonion.net/en/2.3/soup.html#kibana
https://docs.securityonion.net/en/2.3/kibana.html#diagnostic-logging

**Reply by Axtrus** [reply]
What version did you upgrade from?
A) From 2.3.182 and 2.3.130

What are the hardware specs of your system?
A) RAM: 256Gb, AMD EPYC 7282 16-Core Processor, Disk: 2TB for NSM and 2TB for  the system 

Have you reviewed the following?
A) Yes but it didn't help that's why i'm here

**Reply by TotieBash** [reply]
see #8399 and #8335 in summary below fixed it for me.
![image](https://user-images.githubusercontent.com/86748904/206294752-af2413db-9008-4163-9553-23bfc8ca69fc.png)

**Reply by Axtrus** [reply]
Hi Totie,

Thanks for your feedback, i ran the commands you gave and right now i'm just getting the message "Please upgrade your browser" and the logs errors in kibana are different

![image](https://user-images.githubusercontent.com/74593771/206661901-8a19de2a-976f-4625-a886-f9c66d15ef07.png)
![image](https://user-images.githubusercontent.com/74593771/206662616-90cdd546-f002-48a6-bdc3-c7ea43cd889b.png)

Could you know how to fix this ?

**Reply by TotieBash** [reply]
I have seen that in Grafana where if my browser was not on the latest patch it won't open but I don't remember if I have seen it in Kibana.  
Did you already run "so-elastic-auth true"? 
Have you already tried a different browser? I assume you did, try with Edge, Chrome or I use Brave.  Or try Chrome incognito mode.

**Reply by Axtrus** [reply]
Yes i already ran so-elastic-auth true and i got the message Auth is already enabled.
I tried several browsers (chrome, firefox, edge and chrome incognito mode) and all my browsers are up to date but i'm still stuck with this error

**Reply by TotieBash** [reply]
Yeah sorry but my past issue was from 130-to-140 update and my standalone install was a vm on ESXi so I was  leveraging vm snapshots.  I had multiple issue with that patches documented in #8399.  This week I did 181-to-190 and I have no issue with those patches.  I google that error and it I see this japanese link https://www.cnblogs.com/buwuliao/p/16131438.html I use google translate, and it looks like your error is purely on Elastic domain and you might find the fix on Elastic forums.  The japanese link use Edge browser to punch through the error.

**Reply by Axtrus** [reply]
Understood, Thank you so much for your inputs and assistance, i'll check on elastic forum and see how to fix it.
